mapper/orm/pom.xml

```diff
@@ -97,4 +101,40 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>module-descriptors</id>
+            <!--
+                TODO HSEARCH-3274
+                Trying to generate module info, we got some issues on multi-release dependent modules.
+                Invocation of jdeps failed: jdeps -generate-module-info ...
+                Error: byte-buddy-1.9.5.jar is a multi-release jar file but -multi-release option is not set.
```

Comment:
I tried to investigate this too, but it seems it boils down to excluding dependencies as well.
I added a comment.